docs/architecture/cloud-native/cloud-native-data.md

```diff
@@ -0,0 +1,149 @@
+---
```

Comment:
Drop this file.

Reply:
Removed. I see what I did. I merged updated master into my branch (duh). Should have merged my branch  into updated master. What I get for working 12 hours a day! :)